Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -864,6 +873,12 @@ lemma lintegral_le_meas {s : Set α} {f : α → ℝ≥0∞} (hf : ∀ a, f a 
   · simpa [hx] using hf x
   · simpa [hx] using h'f x hx
 
+lemma setLIntegral_le_meas {s t : Set α} (hs : MeasurableSet s)
+    {f : α → ℝ≥0∞} (hf : ∀ a ∈ s, f a ≤ 1)
+    (hf' : ∀ a ∈ s \ t, f a = 0) : ∫⁻ a in s, f a ∂μ ≤ μ t := by
```

Comment:
I'm sorry for the delay. One last request: please assume `∀ a ∈ s, a ∈ t → f a ≤ 1` and `∀ a ∈ s, a ∉ t → f a = 0`. This is equivalent to your assumptions, but those who apply your lemma won't need to deal with the case `a ∈ s, a ∉ t` twice. Thanks!
bors d+